src/main/java/com/butbetter/storage/model/Address.java

```diff
@@ -0,0 +1,142 @@
+package com.butbetter.storage.model;
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+public class Address {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id", insertable = false, updatable = false, nullable = false)
+    private UUID uuid;
+
+    @Column(name = "name", nullable = false)
```

Comment:
Use @NotBlank instead of nullable = false for all the String attributes of this class.
@NotBlank will not allow null, "" or " ". nullable = false will allow "" and " " as parameters

Reply:
Validation is part of service

Reply:
I can added to NotNull

Reply:
ok thats a question we have to solve because, nullable = false is basicaly also validation.
Where the validation takes place has to be discussed